sdk/netapp/Microsoft.Azure.Management.NetApp/src/Generated/NetAppResourceOperationsExtensions.cs

```diff
@@ -100,21 +100,15 @@ public static async Task<CheckAvailabilityResponse> CheckNameAvailabilityAsync(t
             /// The location
             /// </param>
             /// <param name='name'>
-            /// Resource name to verify.
-            /// </param>
-            /// <param name='type'>
-            /// Resource type used for verification. Possible values include:
-            /// 'Microsoft.NetApp/netAppAccounts',
-            /// 'Microsoft.NetApp/netAppAccounts/capacityPools',
-            /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes',
-            /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots'
+            /// File path to verify.
             /// </param>
-            /// <param name='resourceGroup'>
-            /// Resource group name.
+            /// <param name='subnetId'>
+            /// The Azure Resource URI for a delegated subnet. Must have the delegation
+            /// Microsoft.NetApp/volumes
             /// </param>
-            public static CheckAvailabilityResponse CheckFilePathAvailability(this INetAppResourceOperations operations, string location, string name, string type, string resourceGroup)
+            public static CheckAvailabilityResponse CheckFilePathAvailability(this INetAppResourceOperations operations, string location, string name, string subnetId)
```

Comment:
Third instance of the breaking change in CheckFilePathAvailability.   You may be able to fix these by adding a customization to support the old api